code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1278,6 +1278,9 @@
 
 		oldspecies.on_species_loss(src)
 
+	for(var/mob/living/simple_animal/diona/N in contents) // Let nymphs wiggle out
```

Comment:
may this should be scoped to the Diona `on_species_loss` proc instead

Reply:
Any possibility of another species somehow getting dionas otherwise? Just making sure